### PR TITLE
Feat: exclude-activities

### LINKED ISSUE #629: SAP: excluded-activities
**Extended Description**
- As an ASAP developer, in order to ensure included activities for testing are identified, I want to test for an excluded-activities part within the terms and conditions, and excluded-activity part(s) within the excluded-activities part.

**Acceptance Criteria**
- [x] All Schematron assertion messages are declarative statements which affirm the positive test outcome.
- [x] All Schematron assertion diagnostic messages are declarative statements which explain the negative test outcome.
- [x] The Schematron code has no assertion failures when validated using `src/validations/styleguides/sch.sch` using the `basic` phase.
- [x] XSpec unit tests for positive and negative Schematron assertion outcomes accompany all Schematron assertions (where feasible).
- [x] Check for part name='excluded-activities'
- [x] Check for part name (at least one) of 'excluded-activity' as child of part name 'excluded-activities'

**Definition of Done**
 - [x] Acceptance criteria met
 - [x] Unit test coverage of our code > 95%
 - [x] Automated code quality checks passed
 - [x] Security reviewed and reported
 - [x] Reviewed against plain language guidelines
 - [x] Code must be self-documenting
 - [x] No local tech debt
 - [x] Load/performance tests passed – needs to be created/automated
 - [x] Documentation updated
 - [x] Architectural Decision Record completed as necessary for significant design choices
 - [ ] PR reviewed & approved
 - [ ] Source code merged

### PR DESCRIPTION
**Pull request by markXLIX**
Closes #629 
Added asserts, diagnostics, xspec for:
terms-and-conditions/part[@name='excluded-activities']
part[@name='included-activities']/part[@name='excluded-activity']

minor formatting

### CODE REVIEW

File: src/validations/rules/sap.sch [hunk 1]
```diff
@@ -479,6 +487,20 @@
...
+        <sch:rule
+            context="oscal:terms-and-conditions/oscal:part[@name eq 'excluded-activities']">
+            
+            <sch:assert
+                diagnostics="has-roe-excluded-activity-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.19"
+                fedramp:specific="true"
+                id="has-roe-excluded-activity"
+                role="error"
+                test="oscal:part[@name eq 'excluded-activity']">The SAP Rules of Engagement (ROE) excluded activities have one or more excluded
```
Comment (danielnaab) [inline src/validations/rules/sap.sch:500]:
```suggestion
                test="oscal:part[@name eq 'excluded-activity']">The SAP Rules of Engagement (ROE) excluded activities must have one or more excluded
```

**Review by danielnaab — CHANGES_REQUESTED** [review]
just one suggestion

File: src/validations/rules/sap.sch (hunk 1 above)
Comment (markXLIX) [reply src/validations/rules/sap.sch:500]: I saw a couple of these in other issues I am working on and thought "darn, Dan is gonna find these and comment on them in the PR".

Fixed.

**Review by GaryGapinski — APPROVED** [review]
While not in this PR's changes, there are many formatting errors in both  `sap.sch` and `sap.xspec`. Line widths (150), attribute displacement (separate lines, sorted), indentation are not those agreed upon.

Please plan to correctly format your changes after all the PRs are merged.

**Review by danielnaab — APPROVED** [review]

**Pull request merged**